### ISSUE TITLE
v8.10.0 security vulnerabilities

### ISSUE THREAD

**Issue by chrisahl** [IBM/kui#4954]
**Describe the bug**
A security scan of 18.10.0 shows the following vulnerabilities
![image](https://user-images.githubusercontent.com/32707829/85322118-c82ff600-b493-11ea-99a1-61b4732e8384.png)

**Full details at https://github.com/open-cluster-management/backlog/issues/2896**

From my looking at the vulnerabilities, the following updates are needed:
acorn:6.4.0 (Need 6.4.1) 
kind 6.0.2  (Need 6.0.3)

**Comment by starpit** [comment]
thanks! fwiw: except for yargs-parser, these are all dev dependencies.

**Comment by chrisahl** [comment]
@starpit When I run  `npm audit --production` on `v8.10.0` I get 

```
                                                                               
                       === npm audit security report ===                        
                                                                                
# Run  npm update mkdirp --depth 6  to resolve 3 vulnerabilities
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Low           │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ minimist                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ 0ca121ed08d3adf8caac3647ee3f45a6edc75effcaa447737478b726724… │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ 0ca121ed08d3adf8caac3647ee3f45a6edc75effcaa447737478b726724… │
│               │ > node-pty-prebuilt-multiarch > prebuild-install > mkdirp >  │
│               │ minimist                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/1179                            │
└───────────────┴──────────────────────────────────────────────────────────────┘


┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Low           │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ minimist                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ 92774128d3e27653ff12b9396512da58b3f81b6f68e00330e81b47bc006… │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ 92774128d3e27653ff12b9396512da58b3f81b6f68e00330e81b47bc006… │
│               │ > @patternfly/react-core > @patternfly/react-styles >        │
│               │ emotion > babel-plugin-emotion > mkdirp > minimist           │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/1179                            │
└───────────────┴──────────────────────────────────────────────────────────────┘


┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Low           │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ minimist                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ 81dadd3febeada2a5414032ceab74081208ad94a559defe909e1e27a043… │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ 81dadd3febeada2a5414032ceab74081208ad94a559defe909e1e27a043… │
│               │ > @patternfly/react-core > @patternfly/react-styles >        │
│               │ emotion > babel-plugin-emotion > mkdirp > minimist           │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/1179                            │
└───────────────┴──────────────────────────────────────────────────────────────┘


┌──────────────────────────────────────────────────────────────────────────────┐
│                                Manual Review                                 │
│            Some vulnerabilities require your attention to resolve            │
│                                                                              │
│         Visit https://go.npm.me/audit-guide for additional guidance          │
└──────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Low           │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ minimist                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=0.2.1 <1.0.0 || >=1.2.3                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ d57174dd02e23a9c74234464b356c90a7d78e7c3e26f95fed15bddf5ca4… │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ d57174dd02e23a9c74234464b356c90a7d78e7c3e26f95fed15bddf5ca4… │
│               │ > mkdirp > minimist                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/1179                            │
└───────────────┴──────────────────────────────────────────────────────────────┘
found 4 low severity vulnerabilities in 363 scanned packages
  run `npm audit fix` to fix 3 of them.
  1 vulnerability requires manual review. See the full report for details.
```
